Samples/tutorials/excel-tutorial/README.md

```diff
@@ -0,0 +1,98 @@
+
+The tutorial gives step-by-step instructions on how to add functionality alongside explanations as to why code is being added. This sample is best used if you've completed the tutorial and want to start from a more complete project. It's also helpful when debugging problems you encounter during the tutorial.
+
+## Features
+
+This sample features the basics of working with a workbook in Excel. The functions create and interact with a table, make a chart, freeze the header row, and protect the worksheet. The sample also shows how to use a dialog box.
+
+## Applies to
+
+- Excel on Windows
+- Excel on Mac
+- Excel on the web
+
+## Prerequisites
+
+- Office connected to a Microsoft 365 subscription (including Office on the web).
+- [Node.js](https://nodejs.org/) version 16 or greater.
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) version 8 or greater.
+
+## Solution
+
+Solution | Author(s)
+---------|----------
+Learn the basics of Excel add-ins | Microsoft
+
+## Version history
+
+Version  | Date | Comments
+---------| -----| --------
+1.0 | 1-13-2023 | Initial release
+
+## Run the sample
+
+1. Navigate to the root directory of the `My Office Add-in` project.
+
+1. Run `npm install`.
+
+1. Run `npm run build`.
+
+1. Start the local web server and sideload your add-in:
+
+- To test your add-in in Excel, run the following command in the root directory of your project. This starts the local web server (if it's not already running) and opens Excel with your add-in loaded.
+
+  - Windows: `npm start`
+  - Mac: `npm run dev-server`
+
+- To test your add-in in Excel on the web, run the following command in the root directory of the `My Office Add-in` project. When you run this command, the local web server starts. Replace "{url}" with the URL of an Excel document on your OneDrive or a SharePoint library to which you have permissions.
+
+  ```command line
+  npm run start:web -- --document {url}
+  ```
+
+  The following are examples.
+
+  - `npm run start:web -- --document https://contoso.sharepoint.com/:t:/g/EZGxP7ksiE5DuxvY638G798BpuhwluxCMfF1WZQj3VYhYQ?e=F4QM1R`
+  - `npm run start:web -- --document https://1drv.ms/x/s!jkcH7spkM4EGgcZUgqthk4IK3NOypVw?e=Z6G1qp`
+  - `npm run start:web -- --document https://contoso-my.sharepoint-df.com/:t:/p/user/EQda453DNTpFnl1bFPhOVR0BwlrzetbXvnaRYii2lDr_oQ?e=RSccmNP`
+
+  > NOTE: If you are developing on a Mac, enclose the {url} in single quotation marks. Do not do this on Windows.
+
+    If your add-in doesn't sideload in the document, manually sideload it by following the instructions in Manually sideload add-ins to Office on the web.
+
+1. If the add-in task pane isn't already open in Excel, go to the Home tab and choose the **Show Taskpane** button in the ribbon to open it.
```

Comment:
The previous sections cause this numbering to restart. Probably should indent the previous sections.